acceptance/tests/partitions/partitions_connect_test.go

```diff
@@ -115,56 +112,47 @@ func TestPartitions_Connect(t *testing.T) {
 				"dns.enableRedirection": strconv.FormatBool(cfg.EnableTransparentProxy),
 			}
 
-			serverHelmValues := map[string]string{
-				"server.exposeGossipAndRPCPorts": "true",
-			}
+			defaultPartitionHelmValues := make(map[string]string)
```

Comment:
We don't need server's gossip and rpc ports because they don't need to gossip with clients anymore 🎉